templates/tfengine/components/resources/storage_buckets/main.tf

```diff
@@ -21,6 +21,14 @@ module "{{resourceName . "name"}}" {
   project_id = module.project.project_id
   location   = "{{get . "storage_location" $.storage_location}}"
 
+  {{if $labels := merge (get $ "labels") (get . "labels") -}}
```

Comment:
I'm surprised this construction works, TIL.

Reply:
Took me a few tries to get it working :D